doc/changelog.rst

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+Changes in Version 3.12.0.dev0
```

Comment:
Can you remove the .dev0 here? It's just another place that we might forget to update (like I forgot in #476).

Reply:
Done.